tasks_week3_omer_gertler1/src/Zoo.cpp

```diff
@@ -0,0 +1,179 @@
+#include"Zoo.h"
+#include <string>
+using std::cout;
+using std::cin;
+using std::endl;
+ 
+
+Zoo::Zoo()
+{
+	Location loc;
+	loc.setRow(rand() % ROWS);
```

Comment:
you can use a c-tor for the location and, the save 2 lines instead of setting the row an col